### ISSUE TITLE
When the number of nodes is large, the editor window will become stuck

### ISSUE THREAD

**Issue by JusamLiu** [alelievr/NodeGraphProcessor#185]
Hi, When the number of nodes is large, the editor window will become stuck and slowly。It already  can't edit the graph node。and open the example graph asset,it also stuck.
I use the version 2021.1.15 of the unity and the 1.3 of graphprocessor。
Have any idea to resolve the problem？

**Comment by alelievr** [comment]
Yeah, I observed a massive slow down with the latest versions too. Apparently it's related to the PropertyFields in the graph, I'm investigating the issue

**Comment by JusamLiu** [comment]
I have tried to open the graph in the version 2021.2 of unity,It will be a little  better. but also have a little stuck and slowly

**Comment by AByte-L** [comment]
> Yeah, I observed a massive slow down with the latest versions too. Apparently it's related to the PropertyFields in the graph, I'm investigating the issue

Waiting for repair

**Comment by buctbamboo** [comment]
in current graph，copy 100+ string node or gameobject node，the graph opration will stuck and slowly，when save asset，open it again，the graph can use quickly

**Comment by xiaoweitorch** [comment]
When using a large number of PropertyFields, lag is inevitable. This is because the Bind used by the PropertyField is polled for data updates, increasing the strain on the CPU.
My solution was to switch back to a normal BaseField and manually refresh the node when the data is updated to ensure uniformity of data and view.
Please excuse my very poor English, use deepl to translate
